vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -114,12 +114,14 @@ $global:imagesToPull += @(
     # CNS (Container Networking Service) Owner: jaer-tsun (Jaeryn)
     "mcr.microsoft.com/containernetworking/azure-cns:v1.4.52",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.5.35",
+    "mcr.microsoft.com/containernetworking/azure-cns:v1.5.38",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.6.7",
-    "mcr.microsoft.com/containernetworking/azure-cns:v1.6.10",
+    "mcr.microsoft.com/containernetworking/azure-cns:v1.6.11",
     # CNI installer for azure-vnet. Owner: evanbaker
     "mcr.microsoft.com/containernetworking/azure-cni:v1.5.35",
+    "mcr.microsoft.com/containernetworking/azure-cni:v1.5.38",
     "mcr.microsoft.com/containernetworking/azure-cni:v1.6.7",
-    "mcr.microsoft.com/containernetworking/azure-cni:v1.6.10"
+    "mcr.microsoft.com/containernetworking/azure-cni:v1.6.11"
 )
 
```

Comment:
Should be update these versions below as well?
        # Azure CNI v1 (legacy)
        "https://acs-mirror.azureedge.net/azure-cni/v1.5.35/binaries/azure-vnet-cni-windows-amd64-v1.5.35.zip",
        "https://acs-mirror.azureedge.net/azure-cni/v1.6.5/binaries/azure-vnet-cni-windows-amd64-v1.6.5.zip",
        # Azure CNI v2 (pod subnet)
        "https://acs-mirror.azureedge.net/azure-cni/v1.4.54/binaries/azure-vnet-cni-swift-windows-amd64-v1.4.54.zip",
        # Azure CNI for Overlay
        "https://acs-mirror.azureedge.net/azure-cni/v1.4.54/binaries/azure-vnet-cni-overlay-windows-amd64-v1.4.54.zip"

Reply:
Nodesubnet/v1 definitely yes, the v2/overlay ones we can update just to keep them aligned but those CNI exec are actualyl delivered via CNI installer container on Windows. These zips remain because we need the conflist from them, not for the CNI bins (except v1).